src/test/scala/org/constellation/Fixtures.scala

```diff
@@ -116,4 +120,28 @@ object Fixtures {
   def makeDummyTransaction(src: String, dst: String, keyPair: KeyPair) =
     TransactionService.createDummyTransaction(src, dst, keyPair)(TransactionChainService[IO]).unsafeRunSync
 
+  def manuallyCreateTransaction(
+                                 src: String = "",
+                                 dst: String = "",
+                                 oeHash: String = "",
+                                 sbHash: String = "",
+                                 sbSignature: Seq[HashSignature] = Seq.empty,
+                                 amount: Long = 0L,
+                                 edgeLastTxRef: LastTransactionRef = LastTransactionRef.empty,
+                                 fee: Option[Long] = None,
+                                 salt: Long = ScalaRandom.nextLong,
+                                 lastTxRef: LastTransactionRef = LastTransactionRef.empty
```

Comment:
why you have `edgeLastTxRef` and `lastTxRef`? it should be equal

Reply:
It's a function intended to be used for creating arbitrary test data. There where tests where I needed lastTxRef's to be different e.g. to check validation logic.

Reply:
so IMO it should be used **only** for invalid transactions. correct ones should be created using separate fn.

Reply:
I would also use it in cases where we don't necessarily need a correct transaction from the business point of view. Like asserting on feeValue in this PR. Whenever a "valid" transaction is needed we should not rely on this function in tests.

Reply:
If we really aim to only use correct transactions in our codebase than I think we should restrict the creation of transaction to only allow instantiating correct ones (by e.g. smart constructor).